.github/workflows/data.yml

```
@@ -0,0 +1,51 @@
+name: Download records & Push to repository
+
+on:
+  push:
+    branches: ["main"]
+  schedule:
+    # Run at 8:00 on working days
+    - cron: 0 8 * * 0-5
```

```suggestion
  schedule:
    # Run at 8:00 on working days
    - cron: 0 8 * * 0-5
  workflow_dispatch:
```

It would be nice to run this on command too.